hw12.php

```diff
@@ -0,0 +1,214 @@
+<?php
+
+declare(strict_types=1);
+
+error_reporting(E_ALL);
+ini_set('display_errors', '1');
+
+
+// task 1
+//function x() {
+//    return "Hello world";
+//};
+//echo x();
+
+
+
+
+// task 2
+
+//function year( int $days) : bool {
+//    $x = '';
+//    if ($days == 365) {
+//        $x = false;
+//    } elseif ($days == 366) {
+//        $x = true;
+//    }
+//    return $x;
+//}
+//
+//var_dump(year(366));
+
+
+
+// task 3
+
+//function qwe(){
+//    $x = explode(' ', 'Cascading Style Sheets');
+//    $y = '';
+//    foreach ($x as $i){
+//        $y .= $i[0];
+//    }
+//    echo $y;
+//}
+//qwe();
+
+
+
+//// task 4
+//
+//function rev($x){
+//    $y = mb_convert_encoding(strrev(mb_convert_encoding($x, 'UTF-16BE', 'UTF-8')),'UTF-8', 'UTF-16LE');
+//    echo $y;
+//};
+//rev('Привет мир');
+
+
+
+// task 5
+//function easy(int $x) : bool
+//{
+//    $y = ' ';
+//    for ($i = 2; $i < $x; $i++){
+//        if ($x % $i === 0){
+//            $y = false;
+//        } else{
+//            $y = true;
+//        }
+//        break;
+//    }
+//    return $y;
+//};
+//var_dump(easy(17));
+
+
+
+
+//// task 6
+//function lastWord(){
+//    $x = explode(' ', 'jngoiregn engrengo iisfi');
+//    $y = strlen(array_pop($x));
+//    echo $y;
+//}
+//lastWord();
+
+
+//
+//// task 8
+//function anagr($x, $y) : bool
+//{
+//    $z = ' ';
+//   if(count_chars($x) == count_chars($y)){
+//       $z = true;
+//   } else{
+//       $z = false;
+//   }
+//
+//   return $z;
```

Comment:
операция сравнения и так вернет bool
```suggestion
//   return count_chars($x) == count_chars($y);
```